solutions/swb-reference/integration-tests/tests/isolated/projects/listUsersForProject.test.ts

```diff
@@ -173,16 +173,11 @@ describe('list users for project tests', () => {
       }
     });
 
-    test('ensure researcher cannot list users for projects in a different project', async () => {
+    test('ensure unauthenticated user cannot list users for projects', async () => {
```

Comment:
is this replacing the researcher test completely?

Reply:
There was already a test for researcher on line 120.